templates/drupal-11/.github/workflows/build_release.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+name: Create Github Release
+
+permissions:
+  contents: write
+
+env:
+  COMPOSE_USER: runner
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    env:
+      COMPOSER_ALLOW_SUPERUSER: 1
+      APP_ENV: prod
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Composer install
+        run: |
+          docker network create frontend
+          docker compose run --rm --user=root phpfpm composer install --no-dev -o --classmap-authoritative
+          docker compose run --rm --user=root phpfpm composer clear-cache
+
+      - name: Make assets dir
```

Some/most projects may need additional building, e.g. to compile theme assets, so adding a “Build additional assets” step may help us to remember this when updating the template. 

```suggestion
      - name: Build additional assets
        # Build any additional assets here
        run: |
          # docker compose run --rm node npm --prefix /app/web/themes/custom/site-theme install
          # docker compose run --rm node npm --prefix /app/web/themes/custom/site-theme run build
          # docker compose run --rm node rm -fr /app/web/themes/custom/site-theme/node_modules
          
      - name: Make assets dir
```